CMakeLists.txt

```diff
@@ -67,9 +67,9 @@ endif()
 
 if(BUILD_32)
 
-  CMAKE_GENERATOR_PLATFORM=ia32
+  set(CMAKE_GENERATOR_PLATFORM ia32)
 else()
-  CMAKE_GENERATOR_PLATFORM=x64
+  set(CMAKE_GENERATOR_PLATFORM x64)
 endif()
```

Comment:
You're right... I can specify the CMAKE_GENERATOR_PLATFORM as a cmd line argument, and remove BUILD_32 argument.